rollup.config.js

```diff
@@ -57,11 +58,8 @@ const rollup = {
 				{find: 'react', replacement: 'dom-chef'},
 			],
 		}),
-		typescript({
-			compilerOptions: {
-				noEmitOnError: !process.env.ROLLUP_WATCH,
-				module: 'Node16',
-			},
+		sucrase({
+			transforms: ['typescript', 'jsx'],
```

Comment:
```js
exclude: ['node_modules/**'],
transforms: ['typescript', 'jsx'],
```

I prefer the config like this, but the `octicons-plain-react` library has exported `tsx` file, so I can't exclude `node_modules`. If you update the`octicons-plain-react`, I think maybe we can use `rollup-plugin-esbuild` to replace `@rollup/plugin-sucrase`.

Reply:
Given how fast it seems to be, I think it’s good as is. As soon as Vitest switches to Rolldown or [esbuild supports `preserveModules`](https://github.com/evanw/esbuild/issues/708) we might change bundler again anyway.